### ISSUE TITLE
Cannot download "https://github.com/sass/node-sass/releases/download/v4.12.0/win32-x64-83_binding.node":

### ISSUE THREAD

**Issue by ErnestAgel** [johnpapa/vue-getting-started#37]
I just trying to follow the instruct :npm install ,it reported that 
setting up Git hooks
can't find .git directory, skipping Git hooks installation

> node-sass@4.12.0 install D:\Desktop\VUE\vue-getting-started\02-getting-started\end\vue-heroes\node_modules\node-sass
> node scripts/install.js

Downloading binary from https://github.com/sass/node-sass/releases/download/v4.12.0/win32-x64-83_binding.node
Cannot download "https://github.com/sass/node-sass/releases/download/v4.12.0/win32-x64-83_binding.node":

HTTP error 404 Not Found

even i tried to "cnpm install" failed too!

**Comment by jj-fx** [comment]
Yeah it looks like they remove old versions. So I went here https://github.com/sass/node-sass/releases/ and it looks like the latest version is 4.14.1

**Comment by frankbret** [comment]
4.12 is there but bugged.
You need to change this line in package.json :
"node-sass": "^4.12.0",
for 
"node-sass": "^4.14.0",
